code_annotations/base.py

```diff
@@ -69,6 +69,8 @@ def __init__(self, config_file_path, report_path_override, verbosity, source_pat
         self.source_path = source_path_override if source_path_override else raw_config['source_path']
         self.echo("Configured for source path: {}".format(self.source_path))
 
+        self.rst_template = raw_config.get('rst_template')
```

Comment:
Can we get a default/sample template added with this to show how it should look?